src/Symfony/Component/Notifier/Bridge/TurboSms/Tests/TurboSmsTransportTest.php

```diff
@@ -128,10 +109,8 @@ public function testInvalidFrom()
         $this->expectException(LengthException::class);
         $this->expectExceptionMessage('The sender length of a TurboSMS message must not exceed 20 characters.');
 
-        $message = new SmsMessage('380931234567', 'Hello!');
-        $transport = new TurboSmsTransport('authToken', 'abcdefghijklmnopqrstu', $this->createMock(HttpClientInterface::class));
+        new TurboSmsTransport('authToken', 'abcdefghijklmnopqrstu', $this->createMock(HttpClientInterface::class));
```

Comment:
the exception happens already in `TurboSmsTransport::constructor()`, no need to call `send()`